index.js

```diff
@@ -0,0 +1,76 @@
+const fs = require("fs");
+const React = require("react");
+const ReactDOMServer = require("react-dom/server");
+
+const App = require("./app");
+
+const getDirectories = (path, ignore = []) =>
+  fs
+    .readdirSync(path)
+    .filter(a => fs.statSync(`${path}${a}`).isDirectory())
+    .filter(a => !ignore.includes(a));
+
+const getFiles = (path, extension, ignore = []) =>
+  fs
+    .readdirSync(path)
+    .filter(a => fs.statSync(`${path}${a}`).isFile())
+    .filter(a => a.endsWith(extension))
+    .filter(a => !ignore.includes(a))
+    .map(a => {
+      const content = fs.readFileSync(`${path}${a}`).toString();
+      return {
+        name: a,
+        content,
+        headline: content.match(/# \[(.*?)\]/)[1],
+        dateAdded: fs.statSync(`${path}${a}`).birthtime
```

Comment:
`birthtime` 🤔

Reply:
Seemed the most appropriate from [this](https://nodejs.org/docs/latest-v6.x/api/fs.html#fs_stat_time_values) list